test/e2e/util/pod_exec.go

```diff
@@ -95,7 +101,7 @@ func (p *PodExecOptions) Run() error {
 		return err
 	}
 
-	pod, err := client.Pods(p.Namespace).Get(context.TODO(), p.PodName, metav1.GetOptions{})
```

Comment:
TBH `context.TODO()` should not be allowed in code review if we can afford to do it.